decidim-surveys/app/mailers/decidim/surveys/survey_confirmation_mailer.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Surveys
+    class SurveyConfirmationMailer < ApplicationMailer
+      include TranslatableAttributes
+      helper Decidim::SanitizeHelper
+
+      def confirmation(user, questionnaire, answers)
+        @user = user
+        @questionnaire_title = translated_attribute(questionnaire.title)
+        @participatory_space_title = translated_attribute(questionnaire.questionnaire_for.component.participatory_space.title)
+
+        return if answers.blank? || user.nil?
+
+        add_file_with_answers(answers)
+
+        mail(to: "#{@user.name} <#{@user.email}>", subject: t(".subject", questionnaire_title: @questionnaire_title))
+      end
```

Comment:
I have *not* ran it locally, but this should fix the spec failing with `undefined method `host' for nil:NilClass`

```suggestion
      def confirmation(user, questionnaire, answers)
        return if answers.blank? || user.nil?
        
        with_user(user) do 
          @user = user
          @questionnaire_title = translated_attribute(questionnaire.title)
          @participatory_space_title = translated_attribute(questionnaire.questionnaire_for.component.participatory_space.title)
           @organization = user.organization
          add_file_with_answers(answers)
          
          mail(to: "#{@user.name} <#{@user.email}>", subject: t(".subject", questionnaire_title: @questionnaire_title))
        end 
      end
```